float32[] x_next
float32[] x
float32[] mu_model
float32[] obs
float32 dt
---
# result
bool done
